src/pages/try.js

```diff
@@ -21,6 +22,42 @@ if (typeof navigator !== 'undefined') {
     </div>
   )
 }
+
+const examples = [{
+  code: 'C4TwDgpgBMBOHQLxQHIHsAm0CWA7YM80cCUAPlADIQCGAZgNwBQTANhAfAMZQDOArgFsoiJlCh1+uclVp0RAPigAGMePUV0WKADcarftHZ1O2AOYALAogVr16vQegBqKAAoBw48ACUUVx5CULDmVj7MbBxQgiAAKkQiTACQblrQAIx26qmY0ABMWfY52gAssvTldD6FUClpUABslZU+1fbixdAAzDUd9QCszdT0be19uVAA7ENyreEs7ARufqJJntFxCWRKAFK8AHSsaGYMQA',
```

Comment:
let's have the code be in plain text for maintainability, and then convert it w/ js.
e.g.
```
const examples = [{
...
}].map(example => ({name: example.name, code: compress(example.code)}))
```
or something

Reply:
or wait I don't think we need them to be compressed at all?

Reply:
I was originally going to just make these ordinary links, so you could move back and forward in history, but it gatsby doesn't seem to pick up on changes in just the query param. But it made sense to have ti compressed then, now not so much. I'll fix.